ibm/service/dnsservices/resource_ibm_dns_linked_zone_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package dnsservices_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccIBMDNSLinkedZone_basic(t *testing.T) {
+	vpcname := fmt.Sprintf("seczone-vpc-%d", acctest.RandIntRange(10, 100))
+	subnetname := fmt.Sprintf("seczone-subnet-name-%d", acctest.RandIntRange(10, 100))
+	name := fmt.Sprintf("seczone-cr-%s", acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum))
+	description := "new test CR - TF"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() {},
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMDNSLinkedZoneDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMDNSLinkedZoneResource(vpcname, subnetname, acc.ISZoneName, acc.ISCIDR, name, description),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_dns_linked_zone.test", "zone", "seczone-terraform-plugin-test.com"),
+				),
+			},
+		},
+	})
```

Comment:
Extend the test case to add additional test with optional fields/update/ import

Reply:
Added more tests